content/en/docs/howto/chart_releaser_action.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Chart Releaser Action to Automate GitHub Page Charts"
+description: "Describe how to use Chart Releaser Action to automate releasing charts through GitHub pages."
+weight: 1
+aliases: ["/docs/chart_releaser_action/"]
```

Comment:
We usually only need aliases for docs which have been moved, so this line isn't needed since this doc is new.

Reply:
Thanks for pointing out that!
I didn't know the purpose of aliases.